src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/DevTools/RazorSyntaxNode.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.CodeAnalysis.Razor.Protocol.DevTools;
+
+internal sealed class RazorSyntaxNode
```

Comment:
This type name is too easy to cause conflicts, lets rename to SyntaxVisualizerNode